experiment/autobumper/bumper/bumper.go

```diff
@@ -544,13 +544,21 @@ func getVersionsAndCheckConsistency(prefixes []Prefix, images map[string]string)
 	versions := map[string][]string{}
 	for _, prefix := range prefixes {
 		newVersions := 0
+		unbumped_found := false
 		for k, v := range images {
 			if strings.HasPrefix(k, prefix.Prefix) {
 				if _, ok := versions[v]; !ok {
 					newVersions++
 				}
-				versions[v] = append(versions[v], k)
-				if prefix.ConsistentImages && newVersions > 1 {
+				//Only add the bumped images to the new versions map
+				if !strings.Contains(k, v) {
```

Comment:
Why would the image name ever contain the tag? Does `k` include the existing tag (not bump tag) as well?
It would be very helpful to have function comments explaining the format of the parameters. It is difficult to trace the source of the `images` parameter to determine its key and value format.